common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+			FROM = { culture_group = orc_group }			
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Harpy Group
+	tp_guide_eagle_nests = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = harpy_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 300
+		build_time = 270
+		ai_creation_factor = 95
+		
+		retinuesize = 20
+		tech_growth_modifier = 0.2		
+		tradevalue = 15
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+		
+	##Troll Group
+	tp_voodo_charm_vendor = {
+		potential = { 
+			in_trade_route_trigger = yes
+			FROM = { culture_group = troll_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 325
+		build_time = 270
+		ai_creation_factor = 95
+		
+		cultural_techpoints = 0.02	
+		tradevalue = 20
+		trade_route_wealth = 2	
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Dwarfish Group
+	tp_dwarfish_stonemasons = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = dwarf_group }			
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 375
+		build_time = 270
+		ai_creation_factor = 95
+
+		garrison_size = 1.5		
+		retinuesize = 10
+		military_techpoints = 0.01	
+		tradevalue = 20
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Kobolt Group
+	tp_chandlers_guild = {
+		potential = {
+			in_trade_route_trigger = yes
+			FROM = { culture_group = kobolt_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 200
+		build_time = 180
+		ai_creation_factor = 95
+
+		monthly_character_piety = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 20
+		trade_route_wealth = 2			
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Gnomes Group
+	tp_merchant_bots = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = gnome_group }			
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 200
+		build_time = 180
+		ai_creation_factor = 95
+
+		retinuesize = 10
+		military_techpoints = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 10		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Murloc Group
+	tp_murloc_coralmasons = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = murloc_group }			
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 400
+		build_time = 270
+		ai_creation_factor = 95
+
+		tech_growth_modifier = 0.2
+		cultural_techpoints = 0.01
+		tradevalue = 30		

```

Comment:
[The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tradevalue = 25
```